### PR TITLE
Css Refactor

### PR DESCRIPTION
**Pull request by mk-dds**
Pivitol
---
https://www.pivotaltracker.com/story/show/167330353

Description
---
**This css refactor does two main things:**
1) Remove unused css files
2) Refactor repetitive classes to use scss `&` feature

**This refactor does NOT:**
- Check if every single class in each file is used
- Touch the markup at all (we may want a future chore there)
- Rename any classes or aim to make any *actual* changes to styles
- Change any css file structure

ToDo
---
Will squash all refactor commits after approval

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": ".venv/bin/python3.7"
+}
```
[the remark this answers is not in the document]
Comment (mk-dds) [reply .vscode/settings.json]: oops my b!

**Comment by mk-dds** [comment]
@dandds I think I got everything!

**Comment by leigh-mil** [comment]
Still seeing one inconsistency that I missed in my last review!
1. When you toggle the side nav it no longer has a shadow: 
css-refactor:
Screen Shot 2019-07-19 at 2 20 19 PM
atat.code.mil:
Screen Shot 2019-07-19 at 2 20 09 PM

**Comment by mk-dds** [comment]
Got it, @leigh-mil !

**Pull request merged**